great_expectations/dataset/util.py

```diff
@@ -143,6 +154,12 @@ def recursively_convert_to_json_serializable(test_obj):
         ## to the number of digits for which the string representation will equal the float representation
         return [recursively_convert_to_json_serializable(x) for x in test_obj.tolist()]
 
+    #Note: This clause has to come after checking for np.ndarray or we get:
+    #      `ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`
```

Comment:
I'm confused at this change/comment...is the isinstance() call raising that ValueError?

Reply:
No, it's the `test_obj == None`. If test_obj is a numpy array, it attempts to compare it to None and raises the ValueError in the comment.

Reply:
Got it. Weird we hadn't found that earlier.